common/src/main/java/com/wynntils/core/managers/ManagerRegistry.java

```diff
@@ -114,11 +100,9 @@ public static void removeAllFeatureDependency(Feature dependant) {
     }
 
     private static void updateManagerState(Class<? extends Manager> manager) {
```

Comment:
same here. In fact, maybe you should search for "manager" (and "function") in this file and verify that there are no remnants.